sklearn/linear_model/_ridge.py

```diff
@@ -1993,5 +1998,11 @@ def _more_tags(self):
             '_xfail_checks': {
                 'check_sample_weights_invariance':
                 'zero sample_weight is not equivalent to removing samples',
-            }
+                # FIXME: see
+                # https://github.com/scikit-learn/scikit-learn/issues/19858
+                # to track progress to resolve this issue
```

Comment:
Should we focus on this first? 🙂

Reply:
https://github.com/scikit-learn/scikit-learn/pull/19869 will focus on that but I prefer to have the common test firt